src/MICore/CommandFactories/gdb.cs

```diff
@@ -258,5 +263,17 @@ private static string TypeBySize(uint size)
             }
             return TargetArchitecture.Unknown;
         }
+
+        public override async Task Signal(string sig)
```

Comment:
CmdBreakUnix pinvokes a kill. The difference is that you need to do this in remote scenarios and therefore need to go through interpreter exec.